.github/workflows/contracts.yml

```diff
@@ -156,12 +148,21 @@ jobs:
       - name: Install dependencies
         run: npm ci
       
-      - name: Migrate contracts
-        env: 
+      - name: Migrate contracts on Ethereum
+        if: github.event.inputs.environment != 'alfajores'
+        env:
           ETH_HOSTNAME: ${{ secrets.KEEP_TEST_ETH_HOSTNAME }}
           CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY: |
             ${{ secrets.KEEP_TEST_ETH_CONTRACT_OWNER_PRIVATE_KEY }}
         run: npx truffle migrate --reset --network $TRUFFLE_NETWORK
+
+      - name: Migrate contracts on Celo
+        if: github.event.inputs.environment == 'alfajores'
+        env:
+          CELO_HOSTNAME: ${{ secrets.KEEP_TEST_CELO_HOSTNAME }}
+          CONTRACT_OWNER_CELO_ACCOUNT_PRIVATE_KEY: |
+            ${{ secrets.KEEP_TEST_CELO_CONTRACT_OWNER_PRIVATE_KEY }}
+        run: npx truffle migrate --reset --network $TRUFFLE_NETWORK
```

Comment:
We could handle these two in a one step if we update env variables names expected by the truffle configuration (see the other comment).

Reply:
My understanding is that if we'd switch to setting `CHAIN_API_URL` and `CONTRACT_OWNER_ACCOUNT_PRIVATE_KEY` environment variables, we would still need to populate those variables with different values, depending on the network used. Is there a way to do that in one step?
Maybe there is a way to do something like this?:
```
CHAIN_API_URL: if github.event.inputs.environment != 'alfajores' then set ${{ secrets.KEEP_TEST_ETH_HOSTNAME }}; else set ${{ secrets.KEEP_TEST_CELO_HOSTNAME }}; 
```
(I don't know how to do that with [operators supported by GitHub](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#operators))

Reply:
I'm afraid it's not yet supported by GHA 😞  
See: https://github.com/actions/runner/issues/409